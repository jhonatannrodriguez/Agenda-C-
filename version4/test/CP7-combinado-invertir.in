# creo una cola e invierto la prioridad
crearColaPrioridad 10
invertirPrioridad

# creo Persona 1 y agrego a cola
crearAgenda
crearPersona 1 1 Persona1
crearFecha 2/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# invierto la prioridad nuevamente
invertirPrioridad

# verifico que la Persona 1 est√° en la cola y es la prioritaria
estaPersonaEnColaPrioridad 1
imprimirPersonaPrioritaria

# creo Persona 2 y agrego a cola
crearAgenda
crearPersona 2 1 Persona2
crearFecha 10/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 3 y agrego a cola
crearAgenda
crearPersona 3 1 Persona3
crearFecha 9/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 4 y agrego a cola
crearAgenda
crearPersona 4 1 Persona4
crearFecha 8/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# creo Persona 5 y agrego a cola
crearAgenda
crearPersona 5 1 Persona5
crearFecha 7/10/2022
crearEvento 1 Evento1
agregarEventoPersona
imprimirPersona
insertarColaPrioridad

# la persona prioritaria es la 1
imprimirPersonaPrioritaria

# ahora invierto la prioridad y la prioritaria es la 2
invertirPrioridad
imprimirPersonaPrioritaria

liberarColaPrioridad

Fin