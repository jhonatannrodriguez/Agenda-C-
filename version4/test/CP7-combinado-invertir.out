1>#  creo una cola e invierto la prioridad.
2>3>4>#  creo Persona 1 y agrego a cola.
5>6>7>8>9>10>Persona 1: Persona1, 1 años
Evento 1:  Evento1
Fecha: 2/10/2022
11>12>#  invierto la prioridad nuevamente.
13>14>#  verifico que la Persona 1 está en la cola y es la prioritaria.
15>La persona 1 está en la cola de prioridad.
16>Persona 1: Persona1, 1 años
Evento 1:  Evento1
Fecha: 2/10/2022
17>#  creo Persona 2 y agrego a cola.
18>19>20>21>22>23>Persona 2: Persona2, 1 años
Evento 1:  Evento1
Fecha: 10/10/2022
24>25>#  creo Persona 3 y agrego a cola.
26>27>28>29>30>31>Persona 3: Persona3, 1 años
Evento 1:  Evento1
Fecha: 9/10/2022
32>33>#  creo Persona 4 y agrego a cola.
34>35>36>37>38>39>Persona 4: Persona4, 1 años
Evento 1:  Evento1
Fecha: 8/10/2022
40>41>#  creo Persona 5 y agrego a cola.
42>43>44>45>46>47>Persona 5: Persona5, 1 años
Evento 1:  Evento1
Fecha: 7/10/2022
48>49>#  la persona prioritaria es la 1.
50>Persona 1: Persona1, 1 años
Evento 1:  Evento1
Fecha: 2/10/2022
51>#  ahora invierto la prioridad y la prioritaria es la 2.
52>53>Persona 2: Persona2, 1 años
Evento 1:  Evento1
Fecha: 10/10/2022
54>55>Fin.
